### DISCUSSION TITLE
Use 'Marker-Element' as reference point with 'loadVectorUserFunction'

### DISCUSSION THREAD

**Discussion by csas8682** [jgerstmayr/EXUDYN#16] [Q&A on usage of Exudyn] [answered]
Hello, 

I would have another question about the usage of _Exudyn_. 

I am trying to use _Exudyn_ to perform biomechanical analysis, in the course of this i came across the problem that I would need to point a force towards a given reference point (origin of a muscle), which is a fixed point on one body of my multibody system. 

I thought the best way would be, to set a marker element and point the force into its direction. Because if I would do it like this, I do not need any fixed reference points (in the global frame). However untill now I was not able to do it like this, and I was wondering if it is possible in _Edudyn_? 

Thank you once again in advance 🙏

**Comment by jgerstmayr** [comment, marked as answer]
This should be possible with a ConnectorSpringDamper. The action of this connector is always between two markers. The connector has a referenceLength, which could be modified, e.g. by a preStepUserFunction. Alternatively you could also implement your own muscle models in the springForceUserFunction.

**Comment by jgerstmayr** [comment]
Additional Remark: Making loads dependent on the system state (e.g., by reading out the current position of a marker, body or node) is not recommended, as loads are not "differentiated", that is they are not considered in the system jacobian. Therefore the solver would get into trouble. User functions in connectors are considered in the jacobian, so this is the right way to go.
Most general force/torque laws can be implemented with the RigidBodySpringDamper, as it provides the full rigid body information on both sides of markers. But this should not be necessary for the above question.
